optaplanner-benchmarks/optaplanner-perf-framework/src/main/java/org/jboss/qa/brms/performance/examples/nurserostering/NurseRostering.java

```diff
@@ -0,0 +1,61 @@
+package org.jboss.qa.brms.performance.examples.nurserostering;
+
+import java.io.File;
+import java.util.Collections;
+
+import org.jboss.qa.brms.performance.examples.AbstractExample;
+import org.jboss.qa.brms.performance.examples.nurserostering.domain.NurseRoster;
+import org.jboss.qa.brms.performance.examples.nurserostering.domain.ShiftAssignment;
+import org.jboss.qa.brms.performance.examples.nurserostering.persistence.NurseRosteringDao;
+import org.optaplanner.core.api.solver.SolverFactory;
+import org.optaplanner.core.config.score.director.ScoreDirectorFactoryConfig;
+import org.optaplanner.core.config.solver.SolverConfig;
+
+public class NurseRostering extends AbstractExample<NurseRoster> {
+
+    public enum DataSet {
+        LONG("long01.xml"),
+        MEDIUM("medium01.xml"),
+        SPRINT("sprint01.xml");
+
+        private String filename;
+
+        DataSet(String file) {
+            this.filename = file;
+        }
+
+        public String getFilename() {
+            return filename;
+        }
+    }
+
+    private static final String PATH_TO_SOLVER_CONFIG = "/org/jboss/qa/brms/performance/examples/nurserostering/solver/nurseRosteringSolverConfig.xml";
```

Comment:
From the readers' perspective it's preferable to avoid longer lines

Reply:
Fixed.